src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -11,6 +11,7 @@ nginx_sites:
     - name: "{{ deploy_env }}_formplayer"
       hosts: formplayer
       port: "{{ formplayer_port }}"
+      zone: 256k
```

Comment:
Is this to support a larger number of machines?

Reply:
it not accepting the default value 64k and Simon suggested value as 256k

Reply:
Cool, thanks for the explanation